packages/core/src/browser/label-provider.ts

```diff
@@ -12,6 +12,9 @@
  * https://www.gnu.org/software/classpath/license.html.
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * Contributors:
+ *     TypeFox and others - Initial API and implementation
+ *     Jonas Helming <jhelming@eclipsesource.com> Contributed on behalf of STMicroelectronics - Documentation
```

Comment:
The following is not necessary and can be removed (no other reference of similar changes in the sources).
We generally do not modify the headers each time that a new contributor modifies a part of the code they do not have the original license to. The license is only added when the file is first created.

Reply:
removed